assets/model_monitoring/components/src/model_data_collector_preprocessor/run.py

```diff
@@ -268,7 +277,7 @@ def _raw_mdc_uri_folder_to_preprocessed_spark_df(
 
     df = _convert_mltable_to_spark_df(table, preprocessed_input_data, fs, add_tags_func)
     # print("df after converting mltable to spark df:")
-    # df.show()
+    # df.select("data").show(truncate=False)
```

Comment:
do we need this code?

Reply:
no for now, but I'm thinking to keep it for easier trouble shooting, we can ask customer to provide the driver log and study the log.